### PR TITLE
📖 book: Add note about Tiltfile syntax highlighting

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:
Adds documentation about Tiltfile syntax highlighting.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/developer/tilt.md [hunk 1]
```diff
@@ -361,3 +361,8 @@ should monitor for changes. When a dependency is modified, Tilt rebuilds the pro
 machine**, copies the binary to the running container, and executes a restart script. This is significantly faster
 than rebuilding the container image for each change. It also helps keep the size of each development image as small as
 possible (the container images do not need the entire go toolchain, source code, module dependencies, etc.).
+
+## Syntax Highlighting for Tiltfile in Intellij
+
+Syntax highlighting for the Tiltfile in Intellij can be configured with a TextMate Bundle. For instructions, please see:
+[Tiltfile TextMate Bundle](https://github.com/tilt-dev/tiltfile.tmbundle).
```
Comment (sbueringer) [inline docs/book/src/developer/tilt.md:368]: @chrischdi If you've got some time and are interested in this. Can you maybe check if that works with VSCode too? I think it should as there is a TextMate VSCode plugin.

**Comment by sbueringer** [comment]
> /lgtm
> 
> I did a quick search on VSCode textmate bundle support. There are a few packages that add TextMate support, but it didn't look like it could be used with this. Maybe someone can check it out closer than I have.
> 
> Even if this is only an option for the JetBrains users, it would be great to add.

Naadir just told me that for VSCode users the Bazel plugin works and provides auto-linting. I'll update accordingly

File: docs/book/src/developer/tilt.md [hunk 2]
```diff
@@ -361,3 +361,16 @@ should monitor for changes. When a dependency is modified, Tilt rebuilds the pro
 machine**, copies the binary to the running container, and executes a restart script. This is significantly faster
 than rebuilding the container image for each change. It also helps keep the size of each development image as small as
 possible (the container images do not need the entire go toolchain, source code, module dependencies, etc.).
+
+## IDE support for Tiltfile
+
+For Intellij, Syntax highlighting for the Tiltfile can be configured with a TextMate Bundle. For instructions, please see:
+[Tiltfile TextMate Bundle](https://github.com/tilt-dev/tiltfile.tmbundle).
+
```
Comment (sbueringer) [inline docs/book/src/developer/tilt.md:369]: Credit to @randomvariable for the info about the Bazel plugin

**Review by chrischdi — COMMENTED** [review]
/lgtm

Works great :-)

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**